docs/01-overview/main-areas/observability/obsv-10-useful-links.md

```diff
@@ -17,10 +17,10 @@ If you're interested in learning more about the Observability area, check out th
 
 - Troubleshoot Observability-related issues:
   - [Prometheus Istio Server keeps crashing](../../../04-operation-guides/troubleshooting/observability/obsv-01-troubleshoot-prometheus-istio-server-crash-oom.md)
-  - [Jaeger shows fewer traces than expected](../../../04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-jaeger-shows-few-traces.md)
+  - [Tracing backend shows fewer traces than expected](../../../04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md)
```

Comment:
Shall we say "tracing backend" or "trace backend" ? Consistency would be great.
Most other docs say "trace backend", so it's less effort to adjust it here.

```suggestion
  - [Trace backend shows fewer traces than expected](../../../04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md)
```